src/packages/@environments/features/environment-list/layout/EnvironmentList.svelte

```diff
@@ -0,0 +1,255 @@
+      loggedUserRoleInWorkspace,
+      workspaceLevelPermissions.ADD_ENVIRONMENT,
+    );
+  }
+
+  const workspaceLoadingSubscribe = isWorkspaceLoaded.subscribe((value) => {
+    isLoading = !value;
+  });
+
+  onDestroy(() => {
+    workspaceLoadingSubscribe();
+  });
+</script>
+
+<div class={`env-sidebar bg-secondary-900 px-3 py-3`} style={``}>
+  <div
+    class={`d-flex justify-content-between curr-workspace-heading-container mb-2 `}
+  >
+    <p
+      class={`mb-0 text-whiteColor ellipsis text-fs-16`}
+      style={`text-color: #FFF;`}
+    >
+      {currentWorkspace?.name || ""}
+    </p>
+    <Tooltip
+      title={!addEnvDisabled ? `Add Environment` : PERMISSION_NOT_FOUND_TEXT}
+      placement={"left"}
+    >
+      <WithIconButtonV2
+        icon={PlusIcon}
+        disable={!hasWorkpaceLevelPermission(
+          loggedUserRoleInWorkspace,
+          workspaceLevelPermissions.ADD_ENVIRONMENT,
+        )}
+        onClick={() => {
+          onCreateEnvironment(localEnvironment);
+        }}
+        loader={false}
+      />
+    </Tooltip>
+  </div>
+
+  {#if false}
+    <div class="spinner">
+      <Spinner size={`32px`} />
+    </div>
+  {:else}
+    {#if globalEnvironment && globalEnvironment.length > 0}
+      <p
+        role="button"
+        class={`fw-normal env-item text-fs-14 border-radius-2 my-2 ${
+          globalEnvironment[0]?.id === currentEnvironment?.id && "active"
+        }`}
+        on:click={() => {
+          onOpenGlobalEnvironment(globalEnvironment[0]);
+        }}
+      >
+        {globalEnvironment[0]?.name}
+      </p>
+    {/if}
+    <hr class="mb-0 mt-1" />
+
+    <p class="fw-normal text-fs-14 rounded my-2" style="padding-left: 12px;">
+      Environment Variables
+    </p>
+    {#if localEnvironment && localEnvironment.length === 0}
+      <div class={`pt-3 pb-2`}>
+        <p class={`add-env-desc-text text-fs-12 mb-0 fw-normal text-center`}>
+          Add Environments to your Workspace to test your APIs with the relevant
+          set of resources and constraints.
+        </p>
+      </div>
+    {/if}
+    <ul class={`env-side-tab-list p-0`}>
+      <List height={"calc(100vh - 180px)"} classProps={"pb-2 pe-2"}>
+        {#if localEnvironment && localEnvironment.length > 0}
+          {#each localEnvironment as env}
+            <ListItem
+              {env}
+              {currentWorkspace}
+              {currentEnvironment}
+              {onDeleteEnvironment}
+              {onUpdateEnvironment}
+              {onOpenEnvironment}
+              {onSelectEnvironment}
+            />
+          {/each}
+        {/if}
+        <p
+          class="fw-normal d-flex align-items-center text-fs-14 m-2 ms-3"
+          role="button"
+          on:click={() => {
+            onCreateEnvironment(localEnvironment);
+          }}
+        >
+          <PlusIcon />
+          <span>Add Environment</span>
+        </p>
+      </List>
+    </ul>
+  {/if}
+</div>
+
+<style lang="scss">
+  .navbar {
```

Comment:
is this css required? `.navbar` normally is used for navigation bar at top of the page in bootstrap. I don't think we are using `navbar` class as well in the current scope.

Reply:
resolved